versions.props

```diff
@@ -12,6 +12,9 @@ org.apache.commons:commons-lang3 = 3.9
 com.fasterxml.jackson.*:* = 2.10.0
 com.github.ben-manes.caffeine:caffeine = 2.7.0
 org.apache.arrow:arrow-vector = 0.14.1
+org.apache.flink:* = 1.10.0
+com.netflix.spectator:* = 0.103.0
+com.amazonaws:aws-java-sdk-s3 = 1.11.738
```

Comment:
Why does this include a direct dependency on the AWS SDK? File operations should go through a table's `FileIO` instance. That is `Serializable` and can be sent to writer tasks.

Reply:
Addressed